lib/endpoints/class-wp-json-posts-controller.php

```diff
@@ -1124,6 +1124,15 @@ protected function prepare_links( $post ) {
 			);
 		}
 
+		if ( 'attachment' !== $post->post_type ) {
```

Comment:
Do all post types support attachments? Revisions? Menu items?

Or, phrased another way, does core have an explicit way of denoting a post supports attachments?